app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -62,17 +64,25 @@ def stale_encrypted_email?
     encrypted_attributes[:email].stale?
   end
 
-  def phone
-    get_encrypted_attribute(name: :phone, default: nil)
+  ENCRYPTED_ATTRIBUTES.each do |attribute|
```

Comment:
I'm not sure I would go in this direction. 

I think the tradeoff here is duplication vs ease of understanding. 

Given that this is only removing a bit of duplication but is much less easy to understand, I'd vote to keep the duplication!